modules/vsphere-problem-detector-running.adoc

```diff
@@ -10,15 +10,15 @@
 
 You can override the schedule for running the {operator-name} checks and run the checks immediately.
 
-The {operator-name} automatically runs the checks every 8 hours. However, when the Operator starts, it runs the checks immediately. The Operator is started by the Cluster Storage Operator when the Cluster Storage Operator starts and determines that the cluster is running on vSphere. To run the checks immediately, you can scale the {operator-name} to `0` and back to `1` so that it restarts the {operator-name}.
+The {operator-name} automatically runs the checks every 8 hours. However, when the Operator starts, it runs the checks immediately. The Operator is started by the Cluster Storage Operator when the Cluster Storage Operator starts and determines that the cluster is running on vSphere.
```

Comment:
```suggestion
The {operator-name} automatically runs the checks every 1 hours. However, when the Operator starts, it runs the checks immediately. The Operator is started by the Cluster Storage Operator when the Cluster Storage Operator starts and determines that the cluster is running on vSphere.
```
Ignore the comment if you have updated in another PR.

Reply:
This is updated in the other PR.